app/Console/Commands/ListClient.php

```diff
@@ -43,16 +44,19 @@ public function handle()
         $data = [];
         foreach ($clients as $client) {
             $groups = \App\ClientGroup::where('client_id', $client->id)->pluck('root_group')->toArray();
+            $keyModel = \App\WeblingKey::where('client_id', $client->id)->first();
             
             $data[] = [
                 $client->id,
                 $client->name,
                 implode(' ', $groups),
-                $client->created_at
+                $keyModel ? Crypt::decryptString($keyModel->api_key) : '',
```

Comment:
Mal laut gedacht zur Nachvollziehbarkeit:
Das ist in der Konsole und deshalb ist es nicht so schlimm den API-Key plain auszugeben.
Von aussen kann man nicht auf den Server kommen und dort über diesen Command die Keys holen.
Korrekt? (Ein bisschen paranoid muss man schon sein :). )

Reply:
Ja, Paranoia ist in diesem Fall am richtigen Ort :)
Es handelt sich hier wiederum um Konsolen-output, genau. 
Wenn jemand Zugriff auf die Konsole hat, haben wir so oder so verloren, denn dann ist auch das `.env`-File sowie der Encryption-Key auslesbar. Und die Datenbank...